### PR TITLE
Docs about how to use private registry.

### LINKED ISSUE #113: Operator: document how to pull policies from a private registry
We should document that pulling policies from a private registry can be done by using the Policy Server CRD attribute `imagePullSecret`.

### PR DESCRIPTION
**Pull request by jvanz**
## Description
Adds a new doc page explaining how Policy Servers can be configured to download policies from private registries.

Fix #113

### CODE REVIEW

**Review by viccuad — APPROVED** [review]
LGTM!

**Review by flavio — APPROVED** [review]
LGTM, thanks!

**Review by raulcabello — APPROVED** [review]

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+sidebar_label: "Using private registry"
+title: ""
+---
+
+# Configuring Policy Servers to use private registry
+
+It is possible to specify and configure the credentials used to authenticate
+into a private registry where your policies are stored.
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:9]: This sentence is a little ambiguous. We need to be more explicit about what/who we are authenticating.

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+Before configure your Policy Server instance, you need to store the credentials
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:12]:
```suggestion
Before configuring your Policy Server instance, you need to store the credentials
```

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  --docker-username=myuser \
+  --docker-password=mypass123 \
+  --docker-server=myregistry.io
+```
+
+If you want more information about how to create the secret. Please, go to the
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets).
+
+Once you have the secret created, it is necessary to configure the Policy Server
+instance setting the `imagePullSecret` field with the name of the secret created with the
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:28]:
```suggestion
instance by setting the `imagePullSecret` field with the name of the secret created with the
```

**Review by divya-mohan0209 — COMMENTED** [review]
Left a few suggestions, hope you find them helpful!

File: docs/operator-manual/policy-servers/02-private-registry.md (hunk 1 above)
Comment (jvanz) [reply docs/operator-manual/policy-servers/02-private-registry.md]: I've rewrote the paragraph. Can you review again, please?

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+title: ""
+---
+
+# Configuring Policy Servers to use private registry
+
+It is not uncommon that enterprises using cloud native environments have their
+own container registries to store their container images and other artefacts
+used to run their applications. As these artefacts are, most of the time, private
+it does not make sense to let their be public available in the web or even for
+some parts of the same organization. In this scenarios is common the
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:12]:
```suggestion
other teams within the same organization. In these scenarios, it is common to
```

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+use of private registries requiring authentication to access the resources stored
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:13]:
```suggestion
use private registries that require authentication to access the resources stored
```

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+on them. In that in mind, it possible to specify and configure the credentials
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:14]:
```suggestion
In them. With this in mind, we have made it possible to specify and configure the credentials
```

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+used to authenticate into a private registry where your policies are stored.
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:15]:
```suggestion
used to authenticate into a private registry where your policies are stored and
```

File: docs/operator-manual/policy-servers/02-private-registry.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+Allowing Kubewarden Policy Server to download policies from public and private
```
Comment (divya-mohan0209) [inline docs/operator-manual/policy-servers/02-private-registry.md:16]:
```suggestion
allowing Kubewarden Policy Server to download policies from public and private
```

**Review by divya-mohan0209 — COMMENTED** [review]
Just some small changes. After you address them, please feel free to merge!

**Pull request merged**